distribution/docs/src/main/resources/content/_managing/_installing/configure-single-sign-on.adoc

```diff
@@ -0,0 +1,76 @@
+:title: System Configuration Settings
+:type: installing
+:status: published
+:summary: Configure Single Sign On(SSO) page of installer.
+:project: ${branding}
+:order: 07
+
+.[[_configuring_sso]]Configure Single Sign On(SSO)
+****
+Configure Single Sign On method: SAML or OIDC.
+
+.SAML SSO
+Enter the URLs for the IdP metatdata and set other options.
+
+IdP Server::
+Configure ${BRANDING}'s' internal Identity Provider Server.
+
+. *_SP Metadata_*: The metatada of the desired Service Provider to connect to the internal IdP. Can be configured as an HTTPS URL (https://), file URL (file:), or an XML block(`&lt;md:EntityDescriptor&gt;...&lt;/md:EntityDescriptor&gt;`).
```

Comment:
✏️ Another case where there should be a space before the opening paren after `block`, I think?

Reply:
fixed